service/lambdaFunction/logger.js

```diff
@@ -0,0 +1,11 @@
+const loggerGenerator = (token) => {
+    return require('logzio-nodejs').createLogger({
+        token: token,
+        protocol: 'https',
+        host: 'listener.logz.io',
```

Comment:
What happens if the customer account is not in us? looks like the listener address is static

Reply:
Adding option to manage it from UI